src/table.ts

```diff
@@ -719,34 +715,9 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
    * region_tag:bigtable_api_table_readstream
    */
   createReadStream(opts?: GetRowsOptions) {
-    const options = opts || {};
+    const options: GetRowsOptions = opts || {};
     const maxRetries = is.number(this.maxRetries) ? this.maxRetries! : 10;
     let activeRequestStream: AbortableDuplex | null;
-    let rowKeys: string[];
-    let filter: {} | null;
-    const rowsLimit = options.limit || 0;
-    const hasLimit = rowsLimit !== 0;
-    let rowsRead = 0;
-    let numConsecutiveErrors = 0;
-    let numRequestsMade = 0;
-    let retryTimer: NodeJS.Timeout | null;
```

Comment:
rowKeys, rowsLimit, hasLimit, rowsRead are now all tracked in the `ReadRowsResumptionStrategy` object. `numConsecutiveErrors` and `numRequestsMade` are tracked internally in gax as they were previously variables that managed retry logic. `retryTimer` is not used anymore since it was previously used to introduce a delay before making a request, but that is now done in google-gax. `filter` is not used anymore as it was previously just assigned in one place and used in one place and now the assigned value is just used inline.